userge/plugins/utils/voice_call.py

```diff
@@ -518,11 +636,56 @@ async def vc_callback(_, cq: CallbackQuery):
 
         elif "back" in cq.data:
             if BACK_BUTTON_TEXT:
-                CQ_MSG = cq.message
                 await cq.edit_message_text(
                     BACK_BUTTON_TEXT,
                     disable_web_page_preview=True,
                     reply_markup=default_markup()
                 )
             else:
                 await cq.message.delete()
+
+    @userge.bot.on_callback_query(filters.regex(r"vol\((.+)\)"))
+    @check_cq_for_all
+    async def vol_callback(cq: CallbackQuery):
+
+        arg = cq.matches[0].group(1)
+        volume = 0
+
+        if arg.isnumeric():
+            volume = int(arg)
+
+        elif arg == "custom":
+
+            try:
+
+                async with userge.conversation(cq.message.chat.id) as conv:
```

Comment:
you can define user here too.
```suggestion
                async with userge.conversation(cq.message.chat.id, cq.from_user.id) as conv:
```